playbooks/gobig/assign-groups.yml

```diff
@@ -23,12 +24,13 @@
         mesos_set: "{{ master_set | union(slave_set) }}"
 
         mongodb_set: "{{ groups.get(mongodb, []) }}"
+        romanesco_set: "{{ groups.get(romanesco, []) }}"
         rabbitmq_set: "{{ groups.get(rabbitmq, []) }}"
         spark_set: "{{ groups.get(spark, []) | union(mesos_set) }}"
         uvcmetrics_set: "{{ groups.get(uvcmetrics, []) }}"
         zookeeper_set: "{{ groups.get(zookeepers, []) | union(mesos_set) }}"
 
-        nodejs_set: []
+        nodejs_set: "{{ romanesco_set }}"
```

Comment:
This setting is just an artifact of how I broke the PRs up.  The next PRs further amend this line to include girder, for example.  I put in the line in question starting from the nodejs PR since it seemed apropos.